<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorsSensorsWidget</class>
 <widget class="QWidget" name="MotorsSensorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motors and Sensors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="motors">
         <property name="text">
          <string>Motors</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/car.png</normaloff>:/icons/car.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pidTuner">
         <property name="text">
          <string>PID Tuner</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/chart_curve.png</normaloff>:/icons/chart_curve.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="servos">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Servos</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/lightning.png</normaloff>:/icons/lightning.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sensorGraph">
         <property name="text">
          <string>Sensor Graph</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/chart_line.png</normaloff>:/icons/chart_line.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sensorList">
         <property name="text">
          <string>Sensor List</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/text_list_bullets.png</normaloff>:/icons/text_list_bullets.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="camera">
         <property name="text">
          <string>Camera</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/webcam.png</normaloff>:/icons/webcam.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="depth">
         <property name="text">
          <string>Depth</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/coins.png</normaloff>:/icons/coins.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/construction.qrc"/>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
